Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -293,35 +293,50 @@ Status EncodeFieldValueImpl(Writer* writer, const FieldValue& field_value) {
   // TODO(rsgowman): some refactoring is in order... but will wait until after a
   // non-varint, non-fixed-size (i.e. string) type is present before doing so.
   switch (field_value.type()) {
-    case FieldValue::Type::Null:
-      RETURN_IF_ERROR(writer->WriteTag(
-          PB_WT_VARINT, google_firestore_v1beta1_Value_null_value_tag));
-      RETURN_IF_ERROR(writer->WriteNull());
+    case FieldValue::Type::Null: {
```

Comment:
Note: [ABSL_MUST_USE_RESULT](https://github.com/abseil/abseil-cpp/blob/master/absl/base/attributes.h#L397-L422) can be useful to make sure these checks aren't forgotten.

Reply:
You mean on the Status return value? If so, that's already done via ABSL_MUST_USE_RESULT on the Status class itself (so it should apply everywhere.)